fe/src/main/java/org/apache/doris/alter/RollupJobV2.java

```diff
@@ -305,7 +305,9 @@ protected void runWaitingTxnJob() throws AlterCancelException {
         if (db == null) {
             throw new AlterCancelException("Databasee " + dbId + " does not exist");
         }
-        
+
+        // TODO(wangbo): 2020/2/4 in the case that the [WaitingTxnJob|PendingJob] is checkpointed and replayMethod won't be called, make sure the tablet meta exists in olapTable and TabletInvertedIndex before doris run
```

Comment:
what's the meaning of this comment?
After checkpoint, we must replay the journal less than checkPointVersion

Reply:
TabletInvertedIndex and MaterializedIndex in olaptable can't be persisted until job is final,they     
depend on replay to load data to memory

Reply:
I think this comment can be removed when you keep the job's state
in PENDING after calling `replayPending()`. Am I right?